style.css

```diff
@@ -0,0 +1,730 @@
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+}
+
+body {
+  overflow-x: hidden;
+  background: url(assets/header-bg.jpg);
+  background-size: 100%;
+}
+
+p {
+  font-size: 20px;
+}
+
+p.small {
+  font-size: 16px;
+}
+
+.navbar {
+  width: 100%;
+  height: 70px;
+  display: flex;
+  flex-direction: row;
+  padding-top: 10px;
+  justify-content: space-between;
+  position: fixed;
+  background: rgba(242, 242, 242, 0.1);
+  padding-left: 25px;
+  padding-right: 25px;
+}
+
+.navbar .logo-container {
+  display: none;
+}
+
+#toggle-icon {
+  display: block;
+}
+
+#toggle-icon div {
+  width: 35px;
+  height: 3px;
+  background-color: black;
+  margin: 6px 0;
+  transition: 0.4s;
+  align-self: flex-end;
+}
+
+nav .nav-menu {
+  display: none;
+  position: fixed;
+  top: 50px;
+  left: 0;
+  height: calc(100vh - 50px);
+  width: 100%;
+}
+
+.open .nav-menu {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  list-style-type: none;
+  background-color: rgba(211, 211, 211, 0.1);
+  -webkit-backdrop-filter: blur(5px);
+  backdrop-filter: blur(5px);
+  z-index: 1;
+  padding: 30px 30px;
+}
+
+.nav-menu li {
+  margin-bottom: 40px;
+  cursor: pointer;
+}
+
+.nav-menu li a {
+  font-family: "Lato", sans-serif;
+  font-size: 18px;
+}
+
+.open .bar1 {
+  -webkit-transform: rotate(-45deg) translate(-6px, 6px);
+  transform: rotate(-45deg) translate(-6px, 6px);
+}
+
+.open .bar2 {
+  opacity: 0;
+}
+
+.open .bar3 {
+  -webkit-transform: rotate(45deg) translate(-6px, -8px);
+  transform: rotate(45deg) translate(-6px, -8px);
+}
+
+.links-container ul {
+  display: none;
+}
+
+header {
+  padding: 70px 25px 25px 50px;
+}
+
+header .header-container {
+  display: flex;
+  flex-direction: column;
+  margin-top: 35px;
+}
+
+.header-container .welcome-heading {
+  color: #ec5242;
+  font-family: "Lato", sans-serif;
+  font-size: 17px;
+}
+
+.header-container .intro {
+  font-family: "cocogoose-trial", sans-serif;
+  text-transform: uppercase;
+  display: block;
+  margin-top: 30px;
+  color: rgba(236, 82, 66, 1);
+  -webkit-background-clip: text;
+  font-size: 40px;
+  max-width: 500px;
+  font-weight: normal;
+}
+
+.header-container .intro-paragraph {
+  border: 2px solid white;
+  background-color: #d3d3d3;
+  padding: 15px;
+  font-family: "Lato", sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  color: #4a4848;
+  margin-top: 30px;
+  max-width: 500px;
+}
+
+.header-container .date {
+  font-family: "Cocogoose-trial", sans-serif;
+  font-size: 30px;
+  color: #4a4848;
+  margin-top: 25px;
+}
+
+.main-program {
+  background: url(assets/mailing-bg.jpg);
+  padding-top: 25px;
+  padding-bottom: 25px;
+  display: flex;
+  flex-direction: column;
+  max-height: 100vh;
+}
+
+.main-program .main-program-heading {
+  color: white;
+  font-family: "Lato", sans-serif;
+  text-align: center;
+  margin-bottom: 12px;
+}
+
+.main-program-hr {
+  width: 35px;
+  align-self: center;
+  color: #ec5242;
+}
```

Comment:
#### Your Desing 👇 
![image](https://user-images.githubusercontent.com/78251927/225739609-1c68fe4d-9021-4702-a0a5-5d181f23c031.png)

![image](https://user-images.githubusercontent.com/78251927/225739736-1deb5faf-2c24-42ab-8bca-06c7acc17779.png)

![image](https://user-images.githubusercontent.com/78251927/225739847-69cf09eb-8aed-46db-8daa-2e5ff3d4e70c.png)

Please kindly update the color of the `hr` as requested by the previous reviewer to match the design

#### Template 👇 
![image](https://user-images.githubusercontent.com/78251927/225740171-4414088d-79b1-4e60-b31c-a2cdd4920063.png)

![image](https://user-images.githubusercontent.com/78251927/225740213-0bf53497-6409-4a44-83b4-5d167f626c65.png)

![image](https://user-images.githubusercontent.com/78251927/225740273-9ecfca3c-2e7e-402f-9ba9-1c09d52ac19f.png)

Reply:
@abdulhamiid @levy002 regarding the color of the hr, I made sure it is #ec5242, and it shows the right color when I run it from my end both on mobile and desktop view. I don't understand why it shows up with a different color from your end.

Here are some screenshots
![image](https://user-images.githubusercontent.com/46492634/225810448-376244b3-6d13-4df1-b07c-402fa8ba9d65.png)

![image](https://user-images.githubusercontent.com/46492634/225810482-b530629c-0ef9-4810-935f-a15a28063267.png)

![image](https://user-images.githubusercontent.com/46492634/225810528-063a3ae8-44a6-49eb-923d-cd0366f13649.png)

![image](https://user-images.githubusercontent.com/46492634/225810570-68f637b4-9eaa-4f99-9b62-d32d6401150f.png)

![image](https://user-images.githubusercontent.com/46492634/225810603-8a5ecfb4-2f44-4a78-a23c-8617b5a5af59.png)

![image](https://user-images.githubusercontent.com/46492634/225810636-ff17c068-815d-4fdb-ba38-3ae8c5a74509.png)

![image](https://user-images.githubusercontent.com/46492634/225810669-9f53cfe1-f67a-41a3-b99a-58344a14744a.png)

![image](https://user-images.githubusercontent.com/46492634/225810735-3fd3dc15-54c2-4e57-8710-871f3553adfb.png)

I made sure that it looks fine from my end, which is why I requested a review, but I don't know where the problem is so you can see it this way